defi/src/protocols/data4.ts

```diff
@@ -10181,5 +10181,48 @@ const data4: Protocol[] = [
     parentProtocol: "parent#goosefx",
     listedAt: 1743849733
   },
+  {
+    id: "5999",
+    name: "Dogecoin",
+    address: null,
+    symbol: "DOGE",
+    url: "https://dogecoin.com",
+    description: "Dogecoin is a cryptocurrency that was created as a joke, but has since gained a large following and is used for various purposes, including tipping and charitable donations.",
+    chain: "Dogecoin",
+    logo: `${baseIconsUrl}/doge.jpg`,
+    audits: "0",
+    audit_note: null,
+    gecko_id: "dogecoin",
+    cmcId: "74",
+    category: "Chain",
+    chains: ["Dogecoin"],
+    forkedFrom: [],
+    oracles: [],
+    module: "dummy.js",
+    twitter: "dogecoin",
+    audit_links: []
+  },
+  {
+    id: "6000",
+    name: "Official Trump",
+    address: "solana:6p6xgHyF7AeE6TZkSmFsko444wqoP15icUSqi2jfGiPN",
+    symbol: "TRUMP",
+    url: "https://gettrumpmemes.com/",
+    description: "The Official Trump token, also known as TRUMP, is a meme token launched by President-elect Donald Trump on his Truth Social account. It is a Solana meme coin that allows users to trade and invest in it.",
+    chain: "Solana",
+    logo: `${baseIconsUrl}/trump.png`,
```

Comment:
logo should be official-trump.jpg ( we recently try to add only jpg files instead of png)